generators/Generators.fs

```diff
@@ -871,20 +871,37 @@ type PalindromeProducts() =
         sprintf "(%s, %s)" (string value.[0]) (string value.[1])
 
     let toPalindromeProducts (value: JToken) =
-        let palindromeValue = value.Value<int>("value")
+        let palindromeValue = value.Value("value")
         let factors = 
             value.SelectToken("factors")
             |> List.mapRender toFactors
 
-        sprintf "(%d, %s)" palindromeValue factors
+        match palindromeValue with
+        | null -> sprintf "(None, %s)" factors
```

Comment:
I think you should be able to use the `Obj.render` function here.

Reply:
I think we can also go with "(None, [ ])" since it will be always like this in this case

Reply:
Yeah, that would work!